### PR TITLE
[DOCFIX] Correct wrong code example

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/_includes/Tiered-Storage-on-Alluxio/pin-file.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ```java
-AlluxioFile file = AlluxioFileSystem.open("/myFile");
-SetStateOptions pinOpt = new SetStateOptions.Builder(ClientContext.getConf()).setPinned(true);
-AlluxioFileSystem.setState(file, pinOpt);
+AlluxioURI uri = new AlluxioURI("/myFile");
+SetAttributeOptions pinOpt = SetAttributeOptions.defaults().setPinned(true);
+FileSystem.Factory.get().setAttribute(uri, pinOpt);
```
Comment (calvinjia) [inline docs/_includes/Tiered-Storage-on-Alluxio/pin-file.md]: Could you make the example four lines and assign `FileSystem.Factory.get()` to a variable? We don't want to suggest users to always call that to access a `FileSystem` object.

Comment (apc999) [inline docs/_includes/Tiered-Storage-on-Alluxio/pin-file.md]: got it.

**Comment by apc999** [comment]
@calvinjia PTAL

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**